studio/pages/project/[ref]/auth/settings.tsx

```diff
@@ -188,7 +192,7 @@ const Settings = () => {
             checked={model.EXTERNAL_EMAIL_ENABLED}
             descriptionText={authConfig.properties.EXTERNAL_EMAIL_ENABLED.help}
           />
-
+          <NumField name="MAILER_OTP_EXP" step="1" min={0} max={86400}/>
```

Comment:
min of `0` seems very short

Reply:
if it's set to 0, gotrue will default it to 86400 - but yeah i think we should reflect that rather than gotrue doing it behind the scenes

Reply:
maybe we should just impose a min duration like 30seconds for email otps and 10 seconds for sms otps? wdyt?